Tests/app/fixture/AbstractEntityTest.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Tests\app\fixture;
```

Comment:
Are you sure it is the right namespace and place for such a kind `abstract` test? Do we need this test class at all?